### PR TITLE
User management by admins

### PR DESCRIPTION
**Pull request by araspaas**
Add Admin functionality

- Add test for admin to see a merchants name, city, state with buttons to enable or disable
- Add link to the merchants name which goes to the admin merchants index and city and state next to name.
- Add route for a users show page for an admin
- Add test that allows an admin to see all users in the system
- Add view page
- Add users index page for an admin
- Add users variable to users index action and add users show action
- Add test that a admin can see a users show page
- Add a users show page that an admin can see

### CODE REVIEW

File: app/views/admin/dashboard/users_show.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (JBelta) [inline app/views/admin/dashboard/users_show.html.erb:2]: What  is green and has wheels? 

users_show.html.erb:1
...

I lied about the wheels

File: app/controllers/admin/dashboard_controller.rb [hunk 2]
```diff
@@ -10,7 +10,11 @@ def ship
   end
 
   def users_index
+    @users = User.all
+  end
 
+  def users_show
+    @account = User.find(params[:user_id])
```
Comment (Eternal-Flame085) [inline app/controllers/admin/dashboard_controller.rb:17]: @user is banned!!!!!

**Review by Eternal-Flame085 — APPROVED** [review]

File: app/controllers/admin/dashboard_controller.rb (hunk 2 above)
Comment (JBelta) [inline app/controllers/admin/dashboard_controller.rb:17]: @user defined in our ApplicationController,  changed to @account

**Review by JBelta — APPROVED** [review]

**Review by SageOfCode — APPROVED** [review]
This looks to be sufficient.

**Pull request merged**